lms/djangoapps/instructor/enrollment.py

```diff
@@ -266,7 +272,26 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
 
     if delete_module:
         module_to_reset.delete()
-        _fire_score_changed_for_block(course_id, student, block, module_state_key)
+        create_new_user_action_id()
+        grade_update_root_type = 'edx.grades.problem.state_deleted'
+        set_user_action_type(grade_update_root_type)
+        tracker.emit(
+            unicode(grade_update_root_type),
+            {
+                'user_id': unicode(student.id),
+                'course_id': unicode(course_id),
+                'problem_id': unicode(module_state_key),
+                'instructor_username': unicode(requesting_user.username),
```

Comment:
The instructor's username is both easily accessible and more informative to course teams than an ID, I think.